Cyberduck/Profiles.md

```diff
@@ -43,7 +43,10 @@ The following properties can be defined in a connection profile:
 - `OAuth Client ID` For protocols using OAuth 2.0 you can override the registered application client ID with the provider.
 - `OAuth Client Secret` For protocols using OAuth 2.0 you can override the registered application client secret with the provider.
 - `Authorization` Set to `AWS2` to default to AWS2 signature authentication for S3. Default is `AWS4HMACSHA256`.
-- `Properties` List of custom protocol-specific properties.
+- `Properties` List of custom protocol-specific properties. You can set [hidden configuration options](Preferences.html#hidden-configuration-options) for a specific connection profile. Example usages can be found in:
```

Comment:
`./Preferences.md#anchor` instead of `Preferences.html`, makes it easier for docs to auto generate links required.